main.go

```diff
@@ -67,7 +67,20 @@ func assetDelivery(res http.ResponseWriter, r *http.Request) {
 	assetName := r.URL.Path
 	if assetName == "/" {
 		assetName = "/index.html"
+
+		res.Header().Set("Referrer-Policy", "no-referrer")
+		res.Header().Set("X-Frame-Options", "DENY")
+		res.Header().Set("X-Xss-Protection", "1; mode=block")
+
+		// Disable some features that may be abused on an app like this.
+		// https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Feature-Policy
+		res.Header().Set("Feature-Policy", "display-capture 'none'; web-share 'none'")
+
+		// https://developer.mozilla.org/en-US/docs/Web/HTTP/CSP
+		res.Header().Set("Content-Security-Policy", "default-src 'none'; connect-src 'self'; font-src 'self' https://fonts.gstatic.com; img-src 'self'; script-src 'self' 'unsafe-inline'; style-src 'self' 'unsafe-inline' https://fonts.googleapis.com")
```

Comment:
For better readability I'd write that as
```
strings.Join([]string{
"default-src ...",
"connect-src ...",
...
}, "; ")
```

Reply:
What about something like this:

```
const csp = "default-src 'none';" +
  "connect-src 'self';" +
  "font-src 'self' https://fonts.gstatic.com;" +
  "img-src 'self';" +
  "script-src 'self' 'unsafe-inline';" +
  "style-src 'self' 'unsafe-inline' https://fonts.googleapis.com"
```

That should be joined at compile time, I think.

As you mention in #46 that it's possible to remove the external fonts, maybe this PR should be on hold till they're gone.

Reply:
It's better than a huge line though I don't really like string concatenations with `+`… But that's just personal preference so lets roll with the `+` style as it should have lower performance impact.

The #46 change is now available as #47 PR… Would be great if you have a look at the PR but please do not rebase this one ontop of it as I will do a squash-merge (as with every PR) and you would need to figure out that rebase again…

Reply:
I've changed it to a `strings.Join(..)` in a global variable. Is that ok? It has the better readability and it can use the same string instance for every request, which is even better than the `const` I proposed.